src/AS_02_IAB.cpp

```diff
@@ -179,7 +183,7 @@ AS_02::IAB::MXFWriter::OpenWrite(
 
     byte_t clip_buffer[RESERVED_KL_SIZE] = { 0 };
 
-    memcpy(clip_buffer, this->m_Writer->m_Dict->ul(MDD_IMF_IABEssenceClipWrappedElement), ASDCP::SMPTE_UL_LENGTH);
+    memcpy(clip_buffer, element_ul_bytes, ASDCP::SMPTE_UL_LENGTH);
```

Comment:
Why not define MDD_IMF_IABEssenceClipWrappedElement to include 01 and 01 in places of 7f and 7f? This Essence UL is defined to be in this context of OP1 header. What flexibility does defining this UL as 7f and 7f add?

Reply:
> What flexibility does defining this UL as 7f and 7f add?

The UL matches the [SMPTE registers](https://registry.smpte-ra.org/view/development/Essence.xml) exactly.

Reply:
A couple of things:
- It would be good to have a way to set these bytes in ULs in a systematic way instead of using array subscripts; That makes is a supported library functionality
- There will be an issue comparing the UL found in the MXF files back with the MDD table definitions. This happens in as-02-info and klv-walk where the ULs are turned into printable strings for human consumption.

Reply:
Following this expedient pull request I would appreciate a contribution that better addresses the 7f placeholder issues. What constitutes "better" is not yet obvious to me.

Reply:
@jhursty @kblinova Opened https://github.com/cinecert/asdcplib/issues/67